Sources/Reactive.swift

```diff
@@ -0,0 +1,26 @@
+/// Describes a provider of reactive extensions.
+///
+/// - note: `ReactiveExtensionsProvider` does not indicate whether a type is
+///         reactive. It is intended for extensions to types that do not own by
+///         the module so as to avoid name collision and return type ambiguity.
```

Comment:
The wording here is a little off. Should be something like this:

> It is intended for extensions to types that are not owned by the module in order to prevent name collisions and return type ambiguity.